### PR TITLE
F #145: Dependencies for lxd market containers and bug fixes

### PR DESCRIPTION
**Pull request by sergiojvg**

### CODE REVIEW

File: src/datastore_mad/remotes/lxd_downloader.sh [hunk 1]
```diff
@@ -169,6 +169,10 @@ export PATH=\$PATH:/bin:/sbin
 
 echo "nameserver $DNS_SERVER" > /etc/resolv.conf
 
+yum install epel-release openssh-server -y > /dev/null 2>&1
+/usr/bin/echo "centos6" >> /root/version
```
Comment (vholer) [inline src/datastore_mad/remotes/lxd_downloader.sh]: What is the purpose of these `echo centos6` and 7?

File: src/datastore_mad/remotes/lxd_downloader.sh [hunk 2]
```diff
@@ -169,6 +169,10 @@ export PATH=\$PATH:/bin:/sbin
...
+systemctl enable sshd
```
Comment (vholer) [inline src/datastore_mad/remotes/lxd_downloader.sh]: There is no systemd in C6 yet.

File: src/datastore_mad/remotes/lxd_downloader.sh [hunk 3]
```diff
@@ -155,7 +155,7 @@ rm -f /etc/resolv.conf > /dev/null 2>&1
 echo "nameserver $DNS_SERVER" > /etc/resolv.conf
 
 apt-get update > /dev/null
-apt-get install curl -y > /dev/null 2>&1
+apt-get install curl openssh-server -y > /dev/null 2>&1
```
Comment (vholer) [inline src/datastore_mad/remotes/lxd_downloader.sh]: It's good to have DEBIAN_FRONTEND specified for the non-interactive runs. E.g.:
`DEBIAN_FRONTEND=noninteractive apt-get install curl openssh-server -y > /dev/null 2>&1`
￼

File: src/datastore_mad/remotes/lxd_downloader.sh (hunk 1 above)
Comment (sergiojvg) [reply src/datastore_mad/remotes/lxd_downloader.sh]: ups, missed those debug lines

File: src/datastore_mad/remotes/lxd_downloader.sh (hunk 3 above)
Comment (rsmontero) [inline src/datastore_mad/remotes/lxd_downloader.sh]: At the begning of the script there is a configuration section, please add there the extra packages needed:

```
#-------------------------------------------------------------------------------
# Downloader configuration attributes
#-------------------------------------------------------------------------------
...
PACKAGES_DEBIAN_UBUNTU="curl openssh-server"
PACKAGES_CENTOS6="epel-release openssh-server"
```

**Pull request closed without merging**